src/internal/operations/backup.go

```diff
@@ -424,6 +424,13 @@ func (op *BackupOperation) do(
 		lastBackupVersion = mans.MinBackupVersion()
 	}
 
+	// Select an appropriate rate limiter for the service.
```

Comment:
seems like this is a pretty deep spot in the operation process to add configuration.  Why not add it to the top of `.Run()` like the other ctx configs?

Reply:
Thanks! Sounds good.